js/ui/map.js

```diff
@@ -287,6 +287,27 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set constraint on the map's geographical bounds.
```

Comment:
well for zoom operations this is the behavior. for panTo operations it just gets as close as it can to the requested point...